src/main/java/com/textmessenger/service/MessageServiceImpl.java

```diff
@@ -2,20 +2,31 @@
 
 import com.textmessenger.model.entity.Dialog;
 import com.textmessenger.model.entity.Message;
+import com.textmessenger.model.entity.User;
+import com.textmessenger.repository.DialogRepository;
 import com.textmessenger.repository.MessageRepository;
+import com.textmessenger.repository.UserRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
+import java.util.Optional;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Optional'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=120&issues=AWUz3g8PW2AOP0CkBtXa&open=AWUz3g8PW2AOP0CkBtXa)

[ISSUE_KEY]: AWUz3g8PW2AOP0CkBtXa